gobblin-core/src/main/java/gobblin/source/extractor/utils/Utils.java

```diff
@@ -53,7 +53,28 @@ public static String getClause(String clause, String datePredicate) {
     }
     return retStr;
   }
-
+  
+  /**
+   * Get coalesce of columns if there are multiple comma-separated columns
+   *
+   * @param column
+   * @return column
+   */
+  public static String getCoalesceColumnNames(String column) {
+    int columnCount = 0;
```

Comment:
nit: I know this is the original implementation but it might be a good time to fix. It looks a bit expensive to parse the whole string to see if there are one or more commas.

Something along the lines of:

if (null == columnOrColumnList) {
  return null;
} else if (columnOrColumnList.indexOf(',') < 0) {
  return column;
} else {
  return "COALESCE(" + column + ")";
}